templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/service/impl/config/BaseWebSecurityConfig.java

```diff
@@ -16,8 +16,6 @@
 import org.springframework.security.web.authentication.www.BasicAuthenticationFilter;
 import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
 import org.springframework.web.cors.CorsConfiguration;
```

Comment:
Also removed this dead import. You should create the archetype locally and import into Eclipse.
When you get warnings this means you need to rework :)